plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -212,14 +220,14 @@ function auto_sizes_calculate_better_sizes( int $id, $size, string $align, int $
 	return $layout_width;
 }
 
-/**
- * Retrieves the layout width for an alignment defined in theme.json.
- *
- * @since n.e.x.t
- *
- * @param string $alignment The alignment value.
- * @return string The alignment width based.
- */
+	/**
+	 * Retrieves the layout width for an alignment defined in theme.json.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $alignment The alignment value.
+	 * @return string The alignment width based.
+	 */
```

Comment:
```suggestion
/**
 * Retrieves the layout width for an alignment defined in theme.json.
 *
 * @since n.e.x.t
 *
 * @param string $alignment The alignment value.
 * @return string The alignment width based.
 */
```